pyOCD/tools/pyocd.py

```diff
@@ -545,7 +545,7 @@ def do_read(self, args):
         addr = self.convert_value(args[0])
         if len(args) < 2:
             count = 4
-                else:
+        else:
```

Comment:
Thanks for noticing. I had the "compress whitespace" option turned on in Apple's FileMerge that I use for conflict resolution, and I think it was causing these indent issues on some merges. I'm working on a pull request to fix this and add a test script for pyocd-tool.